src/components/preferences/ShortcutsTab.js

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import styles from "./styles";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import { Field } from "formik";
+
+const useStyles = makeStyles(styles);
+
+export default function ShortcutsTab() {
+    const classes = useStyles();
+
+    const onTextChange = (setFieldValue, fieldName) => (event, value) => {
+        const new_value = event.target.value;
+        setFieldValue(fieldName, new_value);
+    };
+
+    const FormTextField = ({
```

Comment:
There's already this exact component in `cyverse-de/ui-lib`.  I'd switch over to using that version of `FormTextField` instead.

Reply:
Yes, use the `FormTextField` from `ui-lib`. That will also mean the `onTextChange` function is no longer needed here, because `Formik` already handles this with its `onChange` function that is part of the `field`, which is spread into the `FormTextField` in `ui-lib`.

The `FormTextField` in `ui-lib` also supports a `label` prop, so these separate `Typography` labels will no longer be needed either.

However, the `inputProps={{ maxLength: 1 }}` prop will need to be added to each `Field` below (that's what that `... custom` prop is for in the `FormTextField` in `ui-lib`).